documentation/release-notes/migration-guide.4.0.0.md

```diff
@@ -171,6 +172,13 @@ Set-AzureRmDiagnosticSettings
 Update-AzureRmDiagnosticSettings
 ```
 
+## Breaking changes to Network cmdlets
+
+The following cmdlets were affected this release:
+
+### New-AzureRmVirtualNetworkGatewayConnection
+- `EnableBgp` parameter has been changed to take a `boolean` instead of a `string`
```

Comment:
Please provide an example of 'Old' usage and 'New' usage as in the rest of the migration guide.